buildcfg.txt

```diff
@@ -801,8 +801,16 @@ M_RDTHIS	0	0
 M_OPTION	0	0
 M_QUITG	0	0
 M_NGAME	0	0
+#ifdef ULTDOOM
+#ifdef DOOM2
```

Comment:
#ifdef doesn't combine in this way. This will remove the M_SKULL lumps from the main Freedoom IWADs.

It's better if you use #else, eg.

#ifdef FREEDM
...
#else 
...
#endif